src/lib/events/duplicateEvent.ts

```
@@ -0,0 +1,65 @@
+import { SupabaseClient } from "@supabase/supabase-js";
+import { addDays, parseISO } from "date-fns";
```

ESLint

[error] 'date-fns' import is restricted from being used. Please import date-fns functions individually from subpaths, e.g. `import format from 'date-fns/format'` instead of destructured imports.

(no-restricted-imports)